.coveragerc

```diff
@@ -1563,7 +1563,8 @@ omit =
     homeassistant/components/zwave_me/switch.py
     homeassistant/components/electrasmart/climate.py
     homeassistant/components/electrasmart/__init__.py
-
+    homeassistant/components/miraie_ac/climate.py
```

Comment:
Can you please insert your lines in the correct sorted position?

We know that it is not correctly sorted but don't make it worst :)